src/plone/restapi/tests/http-examples/registry_get_list.resp

```diff
@@ -188,12 +188,10 @@ Content-Type: application/json
           ], 
           "description": "Paths to folders and collections to link to at the portal root.", 
           "items": {
-            "choices": [], 
             "description": "", 
-            "enum": [], 
-            "enumNames": [], 
             "title": "", 
-            "type": "string"
+            "type": "string", 
+            "vocabulary": "http://localhost:55001/plone/@vocabularies/plone.app.vocabularies.SyndicatableFeedItems"
```

Comment:
I am wondering if we should somehow mark this as a "hypermedia" link in the HATEOAS sense here.

Reply:
To be precice: "vocabulary": { "@id": "http://localhost:55001/plone/@vocabularies/plone.app.vocabularies.SyndicatableFeedItems"}

Reply:
Yes, I agree. We should consider using hypermedia valid structure more.

As for guillotina, the cms package has some very basic support https://github.com/plone/guillotina_cms/blob/158186cc34eef9b70e7f228a1013d1e8da6172b1/guillotina_cms/api/vocabularies.py

but I don't know much about the implementation.